samples/net/openthread/coprocessor/prj.conf

```diff
@@ -0,0 +1,27 @@
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_LOG=y
+
+# Set OpenThread NCP architecture
+CONFIG_OPENTHREAD_COPROCESSOR=y
+CONFIG_OPENTHREAD_COPROCESSOR_NCP=y
+
+# Turn off OpenThread shell to not have conflict with Zephyr shell
+CONFIG_OPENTHREAD_SHELL=n
```

Comment:
Zephyr shell ? Do you mean the `SHELL` setting.
But from what I can see, `OPENTHREAD_SHELL` selects `SHELL`.

If something is conflicting, then it sound like a missing dependency handling in Kconfig.

Reply:
This is really the shell that comes with OpenThread and has been integrated with the Zephyr shell. Command from the OpenThread shell can be called by prefixing with 'ot ', e.g. `ot ipaddr`. If the shell is enabled together with RCP/NCP, then there is a double conflicting UART function definition. Should `CONFIG_OPENTHREAD_COPROCESSOR=y` imply `CONFIG_OPENTHREAD_SHELL=n`?

Reply:
The comment seems to be incorrect, we don't want OpenThread CLI at all in NCP sample (NCP does not support it).

BTW, you've already added dependency in this PR:
```
config OPENTHREAD_SHELL
	bool "Enable OpenThread shell"
	depends on !OPENTHREAD_COPROCESSOR
	select SHELL
	default y
```
So it's not needed anymore to disable `OPENTHREAD_SHELL` in NCP/RCP project config.

Reply:
> Something went wrong with the rebase, a lot of unrelated files showed up in the commit.

Yes, mixed up some stash into the rebase. Fixing it currently.

Reply:
Removed.